components/README.md

```diff
@@ -0,0 +1,3 @@
+# Kwenta Components
```

Comment:
Nice idea to add a README

Reply:
Thanks! I'll definitely start adding more information to this in the next phase.